docs/source/api.js

```diff
@@ -22,6 +22,18 @@ const files = [
   'lib/aggregate.js',
   'lib/cursor/AggregationCursor.js',
   'lib/schematype.js',
+  'lib/schema/array.js',
```

Comment:
IMO we shouldn't add `schema/boolean`, etc. to the API docs. These docs expose a lot of functions that people just don't use, and the distinction between `schema/boolean` and 'boolean' is pretty confusing.

Reply:
That makes sense. I just figured that the @public jsdocs meant they should be in the api docs. Fixing this now.